ui/component/or-mwc-components/src/or-mwc-tabs.ts

```diff
@@ -89,6 +89,7 @@ export class OrMwcTabs extends LitElement {
             if(tabBarElement != null) {
                 this.mdcTabBar = new MDCTabBar(tabBarElement); // init of material component
                 this.mdcTabBar.listen("MDCTabBar:activated", (event: CustomEvent) => {
+                    console.log("MDCTabBar:activated event triggered!");
```

Comment:
Can remove this

Reply:
For now left it in there for testing purposes of fixing or-mwc-tabs issue ( #746 )
Will fix of course before merge.

Reply:
Removed.